src/realmGeol.ttl

```diff
@@ -57,307 +288,1624 @@ sorelsp:hasUpperBoundary rdf:type owl:ObjectProperty .
+                      skos:prefLabel "crustal element"@en .
 
 
 ###  http://sweetontology.net/realmGeol/DLayer
 soreag:DLayer rdf:type owl:Class ;
-            rdfs:subClassOf soreag:Mantle ;
-            rdfs:label "d layer"@en .
+              rdfs:subClassOf soreag:Mantle ;
+              rdfs:label "d layer"@en .
 
 
-###  http://sweetontology.net/realmGeol/Dike
-soreag:Dike rdf:type owl:Class ;
-          rdfs:subClassOf soreag:PlutonicStructure ;
-          rdfs:label "dike"@en .
+###  http://sweetontology.net/realmGeol/DenudationalFeature
+soreag:DenudationalFeature rdf:type owl:Class ;
+                           rdfs:subClassOf soreag:StructuralFeature ;
+                           dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                           dcterms:source <https://doi.org/10.1080/11035897.2016.1178666> ;
+                           skos:definition "Structures shaped by weathering and erosion; include inlier, outlier, window, klippe, unconformity and peneplain (Kumpulainen, 2017)"@en ;
+                           skos:prefLabel "denudational feature"@en .
+
 
+###  http://sweetontology.net/realmGeol/Depression
+soreag:Depression rdf:type owl:Class ;
+                  rdfs:subClassOf soreag:StructuralFeature ;
+                  dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                  dcterms:source <http://www.glossaryofgeology.org> ;
+                  skos:definition "A structurally low area formed in the crust, with various mechanisms of formation. (Modified from Neuendorf et al., 2011)"@en ;
+                  skos:prefLabel "depression"@en .
 
-###  http://sweetontology.net/realmGeol/Fabric
-soreag:Fabric rdf:type owl:Class ;
-            rdfs:subClassOf soprop:Configuration ;
-            rdfs:label "fabric"@en .
+
+###  http://sweetontology.net/realmGeol/Dike
+soreag:Dike rdf:type owl:Class ;
+            rdfs:subClassOf soreag:PlutonicStructure ;
+            rdfs:label "dike"@en .
+
+
+###  http://sweetontology.net/realmGeol/Dome
+soreag:Dome rdf:type owl:Class ;
+            rdfs:subClassOf soreag:StructuralFeature ;
+            dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+            dcterms:source <http://www.glossaryofgeology.org> ;
+            skos:definition "An uplift or anticlinal structure, either circular or elliptical in outline, in which the rocks dip gently away in all directions. A dome may be small or many kilometers in diameter. Domes include diapirs, volcanic domes, and cratonic uplifts. (Neuendorf et al., 2011)"@en ;
```

Comment:
What about those dome-and-basin provinces that have quite steep to subvertical dips on the flanks of domes. A (structural) dome is also a geometrical term, describing a geological feature related to a type-1 interference pattern, being the culmination of a non-cylindrical antiform or the superposition of two antiforms to form a four way closure. Yes two superposed anticlines form the majority of structural domes, but it would still be a dome if there was no established stratigraphic order (two superposed antiforms), or if the rocks were overturned (two synclinal antiforms).